webauthn.cabal

```diff
@@ -41,6 +41,7 @@ extra-source-files:
   changelog.md,
   root-certs/**/*.crt,
   tests/golden-metadata/big/blob.jwt,
+  tests/golden-metadata/big/blob-with-errors.jwt,
```

Comment:
Would it not make sense to just get everything from the golden-metadata directory?

``golden-metadata/**/*.jwt``

Reply:
That might prevent issues like this in the future?

Reply:
The wildcards are unfortunately [very limited](https://cabal.readthedocs.io/en/3.4/cabal-package.html#pkg-field-extra-source-files), I wish we could do `tests/golden-metadata/*`. But yeah `golden-metadata/**/*.jwt` works, I'll change this